Resources/Prototypes/Catalog/Fills/Lockers/dressers.yml

```diff
@@ -70,11 +70,11 @@
       - id: ClothingHeadHatBeretHoS
       - id: ClothingHeadHatHoshat
       - id: ClothingNeckCloakHos
-      - id: ClothingUniformJumpskirtHoSAlt
-      - id: ClothingUniformJumpsuitHoSAlt
+#      - id: ClothingUniformJumpskirtHoSAlt # DeltaV - removed for incongruence
+#      - id: ClothingUniformJumpsuitHoSAlt # DeltaV - removed for incongruence
       - id: ClothingUniformJumpskirtHosFormal
       - id: ClothingUniformJumpsuitHosFormal
-      - id: ClothingOuterWinterHoS
+#      - id: ClothingOuterWinterHoS # DeltaV - removed for incongruence
```

Comment:
I don't see a reason for removing the winter coat from the dresser. It's meant to store clothes. Don't punish the player for forgetting to select the coat in the loadout.

Reply:
Addressed by adei and replied there